store/src/java/com/zimbra/cs/service/admin/SendMdmNotificationEmail.java

```diff
@@ -1,70 +0,0 @@
-/*
- * ***** BEGIN LICENSE BLOCK *****
- * Zimbra Collaboration Suite Server
- * Copyright (C) 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2013, 2014, 2015, 2016, 2022 Synacor, Inc.
- *
- * This program is free software: you can redistribute it and/or modify it under
- * the terms of the GNU General Public License as published by the Free Software Foundation,
- * version 2 of the License.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
- * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
- * See the GNU General Public License for more details.
- * You should have received a copy of the GNU General Public License along with this program.
- * If not, see <https://www.gnu.org/licenses/>.
- * ***** END LICENSE BLOCK *****
- */
-
-package com.zimbra.cs.service.admin;
-
-import com.zimbra.common.service.ServiceException;
-import com.zimbra.common.soap.AdminConstants;
-import com.zimbra.common.soap.Element;
-import com.zimbra.common.soap.SyncAdminConstants;
-import com.zimbra.common.util.L10nUtil;
-import com.zimbra.common.util.ZimbraLog;
-import com.zimbra.cs.account.Provisioning;
-import com.zimbra.cs.account.Server;
-import com.zimbra.cs.rmgmt.RemoteManager;
-import com.zimbra.cs.rmgmt.RemoteResult;
-import com.zimbra.cs.util.AccountUtil;
-import com.zimbra.soap.ZimbraSoapContext;
-
-import java.nio.charset.Charset;
-import java.util.Map;
-
-
-public class SendMdmNotificationEmail extends AdminDocumentHandler {
-
-    @Override
-    public Element handle(Element request, Map<String, Object> context) throws ServiceException {
-        ZimbraSoapContext zsc = getZimbraSoapContext(context);
-        Provisioning prov = Provisioning.getInstance();
-        Server localServer = prov.getLocalServer();
-        AccountUtil.isAdminAccount(getAuthenticatedAccount(zsc));
-        String message = request.getElement(SyncAdminConstants.E_MESSAGE).getText();
-        String status = request.getElement(SyncAdminConstants.E_STATUS).getText();
-        RemoteManager remoteManager = RemoteManager.getRemoteManager(prov.getLocalServer());
-        String command = "zmmdmnotificationmail \"" + message + "\" \"" + status + "\"";
```

Comment:
we are going to remove `zmmdmnotificationmail` also right?

Reply:
should be not required anymore

Reply:
yes so we should remove it from zm-build and zm-core-utils

Reply:
done
https://github.com/Zimbra/zm-build/pull/233
https://github.com/Zimbra/zm-core-utils/pull/124